lib/auth/tun.go

```diff
@@ -39,6 +39,14 @@ import (
 	"golang.org/x/crypto/ssh/agent"
 )
 
+// dialRetryInterval specifies the time interval tun client waits to retry
+// dialing the same auth server
+const dialRetryInterval = time.Duration(time.Millisecond * 50)
```

Comment:
ughh... `defaults` were supposed to be _configuration_ defaults. In other words, if a value is not present in `teleport.yaml`, it should not be in `defaults.go`.

And then we turned it into a kitchen sink for every constant ever mentioned in the code...